Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/nhinc/dao/CorrelatedIdentifiersDaoImpl.java

```diff
@@ -40,13 +40,15 @@ public void addPatientCorrelation(CorrelatedIdentifiers correlatedIdentifers) {
     @Override
     public List<QualifiedPatientIdentifier> retrievePatientCorrelation(
             QualifiedPatientIdentifier qualifiedPatientIdentifier, List<String> includeOnlyAssigningAuthorities) {
-        return Retriever.retrievePatientCorrelation(qualifiedPatientIdentifier, includeOnlyAssigningAuthorities);
+        Retriever retriever = new Retriever();
+        return retriever.retrievePatientCorrelation(qualifiedPatientIdentifier, includeOnlyAssigningAuthorities);
```

Comment:
we can just make this call inline. 
new Retriever().retrievePatientCorrelation(qualifiedPatientIdentifier, includeOnlyAssigningAuthorities);